docs/getting_started/Datus_tutorial.md

```
@@ -0,0 +1,257 @@
+Start the guided tutorial:
+
+```bash
+datus-agent tutorial
+```
+
+You will see a structured 5-step workflow. This will take approximately 10 minutes to initialize through multi-turn agent calls. 
+
+
+### 🔹 Step [1/5] Validate Data & Configuration
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)